CIP-????/README.md

```diff
@@ -0,0 +1,205 @@
+---
+CIP: ????
```

Comment:
```suggestion
CIP: ?
```
Would you mind reordering the header to match the [CIP template](https://github.com/cardano-foundation/CIPs/blob/master/.github/CIP-TEMPLATE.md?plain=1#L1).